### PR TITLE
Closes #17: Add Vimeo icon

### LINKED ISSUE #17: Add Vimeo icon
## Motivation
We're using az-icons via Quickstart 2 and we have a Vimeo account we want to link to.

### Is your feature request related to a problem? Please describe.
Vimeo is sad:
image

### Describe the solution you'd like
Make a Vimeo icon available in this repository. There's a Vimeo icon [in this footer](https://libguides.library.arizona.edu/library-guides) we could use maybe.

### Describe alternatives you've considered
- Not have an icon for Vimeo :-(

### PR DESCRIPTION
**Pull request by MarinMaksutaj**
### Related issue
#17

### CODE REVIEW

File: demo.html [hunk 1]
```diff
@@ -2,13 +2,13 @@
 <html>
 <head>
     <meta charset="utf-8">
-    <title>AZ Icons Demo</title>
+    <title>IcoMoon Demo</title>
```
Comment (trackleft) [inline demo.html:5]:
```suggestion
    <title>AZ Icons Demo</title>
```

File: fonts/az-icons.svg [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <svg xmlns="http://www.w3.org/2000/svg">
 <metadata>Generated by IcoMoon</metadata>
 <defs>
-<font id="az-icons" horiz-adv-x="1024">
+<font id="icomoon" horiz-adv-x="1024">
```
Comment (trackleft) [inline fonts/az-icons.svg:6]:
```suggestion
<font id="az-icons" horiz-adv-x="1024">
```

File: demo.html [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
     <meta name="description" content="An Icon Font Generated By IcoMoon.io">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="az-icons-styles.css"></head>
 <body>
     <div class="bgc1 clearfix">
-        <h1 class="mhmm mvm"><span class="fgc1">Font Name:</span> icomoon <small class="fgc1">(Glyphs:&nbsp;19)</small></h1>
+        <h1 class="mhmm mvm"><span class="fgc1">Font Name:</span> icomoon <small class="fgc1">(Glyphs:&nbsp;20)</small></h1>
```
Comment (trackleft) [inline demo.html:11]:
```suggestion
        <h1 class="mhmm mvm"><span class="fgc1">Font Name:</span> az-icons <small class="fgc1">(Glyphs:&nbsp;20)</small></h1>
```

File: svg/vimeo.svg [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<!-- Generated by IcoMoon.io -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="1024" height="1024" viewBox="0 0 1024 1024">
+<title>youtube</title>
```
Comment (trackleft) [inline svg/vimeo.svg:3]:
```suggestion
<title>vimeo</title>
```

File: svg/vimeo.svg [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+<g id="icomoon-ignore">
```
Comment (trackleft) [inline svg/vimeo.svg:4]:
```suggestion
```

File: svg/vimeo.svg [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+</g>
```
Comment (trackleft) [inline svg/vimeo.svg:5]:
```suggestion
<g id="icomoon-ignore">
</g>
```

**Comment by trackleft** [comment]
My suspicion is that the az-icons font name wasn't entered into ico-moon and so the woff, eot and ttf files most likely reference icomoon as the font name, but I could be wrong.

**Comment by michaelhagedon** [comment]
> My suspicion is that the az-icons font name wasn't entered into ico-moon and so the woff, eot and ttf files most likely reference icomoon as the font name, but I could be wrong.

Thanks @trackleft, that sounds right... we'll regenerate.

**Comment by MarinMaksutaj** [comment]
Sorry about that! Tried to regenerate the files. Hopefully now it's properly showing az-icons everywhere.

**Review by trackleft — APPROVED** [review]

**Comment by trackleft** [comment]
Kevin Cooper to Everyone (10:24 AM)
https://www.virustotal.com/gui/url/035a3ce0aa8b6870d9676c9dd8f67ef21dee8ba4ce405999e07a3f645cd1f1d9?nocache=1
https://www.virustotal.com/gui/url/881c27b34b2069b1a1d8ce854978b6c7e47dbbde5bd0f6bb1c0fde2d7c632ba5?nocache=1
Kevin Cooper to Everyone (10:24 AM)
https://www.virustotal.com/gui/url/49e4b548762581b8da9149a057455b8870e1fa007aff3c4bccaea23ff3037b1b?nocache=1

**Review by kevcooper — APPROVED** [review]
Binary files have been reviewed with virustotal

**Comment by michaelhagedon** [comment]
Oh, no problem, scanning is wise!

**Pull request merged**